NDTensors/src/lib/GradedAxes/src/gradedunitrange.jl

```diff
@@ -17,31 +17,51 @@ using BlockArrays:
   blocklengths,
   findblock,
   findblockindex,
-  mortar
+  mortar,
+  sortedunion
 using Compat: allequal
 using FillArrays: Fill
 using ..LabelledNumbers:
   LabelledNumbers,
   LabelledInteger,
   LabelledUnitRange,
   label,
+  label_type,
   labelled,
   labelled_isequal,
   unlabel
 
-const AbstractGradedUnitRange{T<:LabelledInteger} = AbstractBlockedUnitRange{T}
+abstract type AbstractGradedUnitRange{T,CS} <: AbstractBlockedUnitRange{T,CS} end
```

Comment:
```suggestion
abstract type AbstractGradedUnitRange{T,BlockLasts} <: AbstractBlockedUnitRange{T,BlockLasts} end
```
since that's what we use elsewhere in `GradedAxes`. I know I suggested `CS` previously but I was just copying that from `BlockArrays.jl` and I don't know why they use that, it looks very mysterious to me.

Reply:
Indeed, I did not understand what `CS` means.